src/settings/MappingProfiles/CheckboxGroupField/CheckboxGroupField.js

```diff
@@ -26,7 +25,7 @@ export const CheckboxGroupField = memo(({
             value={option.value}
             isEqual={isEqual}
             render={fieldProps => (
-              <Row>
+              <div className={css.checkboxWrap}>
```

Comment:
Can we use the Layout component for that?

Reply:
Done.